e2e-tests/tfm/cypress/e2e/journeys/case-amendments/add-bank-decision-proceed.spec.js

```diff
@@ -513,7 +507,7 @@ context('Amendments underwriting - add banks decision - proceed', () => {
 
     cy.visit(relative(`/case/${dealId}/deal`));
     caseDealPage.dealFacilitiesTable.row(facilityId).facilityTenor().contains(updatedFacilityTenor);
-    caseDealPage.dealFacilitiesTable.row(facilityId).facilityEndDate().contains(dateConstants.tomorrowFormattedFull);
+    caseDealPage.dealFacilitiesTable.row(facilityId).facilityCoverEndDate().contains(dateConstants.tomorrowFormattedFull);
```

Comment:
could tomorrowFormattedFull be dereferenced at the top? `const { tomorrowFormattedFull } = dateConstants;` as it is used more than once

Reply:
Sure, have changed

Reply:
destructured, will be visible next time i request a review